SpatialOS/scripts/generate-schema-descriptor.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e -x
+pushd "../"
+
+BUILD_DIR="$1"
+PACKAGES_DIR="$2"
```

Comment:
Could this be in a global config somewhere so that this file can be called standalone more easily?